manifests/repo.pp

```diff
@@ -32,15 +32,14 @@
 
   case $::osfamily {
     'Debian': {
-      if !defined(Class['apt']) {
-        class { 'apt': }
-      }
+      include ::apt
+      Class['apt::update'] -> Package<| |>
 
       apt::source { 'elasticsearch':
         location    => "http://packages.elastic.co/elasticsearch/${elasticsearch::repo_version}/debian",
         release     => 'stable',
         repos       => 'main',
-        key         => 'D88E42B4',
+        key         => '46095ACC8548582C1A2699A9D27D666CD88E42B4',
```

Comment:
Which versions of the apt module need to be supported? I'm not a PE user so I'm not familiar with it, though I do know that 1.8 supports the 40 character key ID.